print-nfs.c

```diff
@@ -1803,8 +1807,14 @@ interp_reply(netdissect_options *ndo,
 
 	case NFSPROC_COMMIT:
 		dp = parserep(ndo, rp, length);
-		if (dp != NULL && parsewccres(ndo, dp, ndo->ndo_vflag) != 0)
+		if (dp != NULL && (dp = parsewccres(ndo, dp, ndo->ndo_vflag)) != 0) {
+			if (ndo->ndo_vflag > 1) {
+				/* write-verf-cookie */
+				ND_TCHECK_8(dp);
+				ND_PRINT(" verf %" PRIx64, EXTRACT_BE_U_8(dp));
```

Comment:
Likewise, here `dp` is an argument to `parsewccres()` and a pointer to the cookie, is this intentional?